packages/search-ui/src/hooks/usePosNegTracking.ts

```diff
@@ -0,0 +1,43 @@
+import { ClickTracking, PosNegTracking, ResultClickedFn, Token } from '@sajari/react-hooks';
+import { isFunction, noop } from '@sajari/react-sdk-utils';
+import { PosNegLocalStorageManager } from '@sajari/sdk-js';
+
+import { ResultValues } from '../Results/types';
+
+export interface UseNTrackingParams {
+  token: Token | undefined;
+  tracking: ClickTracking | PosNegTracking | undefined;
+  values: ResultValues;
+  onClick?: ResultClickedFn;
+  posNegLocalStorageManager: PosNegLocalStorageManager;
+}
+
+function usePosNegTracking(params: UseNTrackingParams) {
+  const { onClick, token, tracking, values, posNegLocalStorageManager } = params;
+
+  if (!tracking || tracking instanceof ClickTracking) {
```

Comment:
I'm not sure if it's worth but I guess it'd be safer to depend on the PosNegTracking here since we might have another type of Tracking? 
```suggestion
  if (!tracking || !(tracking instanceof PosNegTracking)) {
```